src/history/hs_cursor.c

```diff
@@ -51,6 +51,10 @@ __wt_hs_row_search(WT_CURSOR_BTREE *hs_cbt, WT_ITEM *srch_key, bool insert)
 #ifdef HAVE_DIAGNOSTIC
     WT_TRET(__wt_cursor_key_order_init(hs_cbt));
 #endif
+    if (ret == 0)
```

Comment:
Create a separate ticket for this bug fix and move this change to the corresponding PR>

Reply:
OK then there are two bug tickets.